cms/envs/common.py

```diff
@@ -1344,7 +1344,7 @@
 UPLOAD_CHUNK_SIZE_IN_MB = 10
 
 ### Max size of asset uploads to GridFS
-MAX_ASSET_UPLOAD_FILE_SIZE_IN_MB = 50
+MAX_ASSET_UPLOAD_FILE_SIZE_IN_MB = 500
```

Comment:
Please make sure that this config value will not be overridden by build configurations. 
I would suggest making this change through configurations instead of through the `common.py` file.

Also, should we make this config change in LMS too?

Reply:
No, this is not overridden by build configuration because recently I have updated this size from 10MB to 50MB and it worked. Further lms don't has this configuration.

Reply:
Could you please confirm this with DevOps?

Reply:
Devops team saying that build configuration don't have any impact on this.